client/package.json

```diff
@@ -56,6 +56,7 @@
   },
   "dependencies": {
     "nib-converter": "^2.2.5",
-    "nib-core": "^2.3.0"
+    "nib-core": "^2.3.0",
+    "query-string": "^6.8.3"
```

Comment:
Per what @gnarf was saying about compiled dist code and dependencies, query-string (and the other nib ones) should be dev deps

Reply:
Yep, I moved these :)